packages/toolkit/scripts/build.ts

```diff
@@ -97,18 +97,21 @@ const entryPoints: EntryPointOptions[] = [
     folder: '',
     entryPoint: 'src/index.ts',
     extractionConfig: 'api-extractor.json',
+    globalName: 'RTK',
   },
   {
     prefix: 'rtk-query',
     folder: 'query',
     entryPoint: 'src/query/index.ts',
     extractionConfig: 'api-extractor.query.json',
+    globalName: 'RTK.QUERY',
```

Comment:
:hand: Does this actually work? I think it would be safer/better to go with a `CONSTANT_CASE` name instead

Reply:
Yes this works, it will generate this line in the UMD file: `(global = typeof globalThis !== 'undefined' ? globalThis : global || self, factory((global.RTK = global.RTK || {}, global.RTK.QUERY = {})));`

Shall I keep it and just make the same change for RTKQ-React or do you want me to rename it as a new global object like `RTKQ`?

Reply:
Let's make both of these be `window.RTKQ`.

Reply:
I've pushed new code to the PR